classes/class-object-sync-sf-salesforce.php

```diff
@@ -835,6 +845,11 @@ public function request_token( $code ) {
 			'redirect_uri'  => $this->callback_url,
 		);
 
+		if ( get_option( $this->option_prefix . 'pkce_support', false ) ) {
+			$challenge = $this->get_challenge();
+			$data['code_verifier'] = $challenge['verifier'] || '';
```

Comment:
Same here re. using `||`, that's not the approach in PHP.

```suggestion
			$data['code_verifier'] = $challenge['verifier'] ?? '';
```